networks/cardano/tokens/931c7c94afa5afe4d86fe6f76c2323c467758b8ffaa73b1f31adb28642616279536e656b.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "",
+    "decimals": 0,
+    "mldId": "cardano-BABYSNEK-931c7c94afa5afe4d86fe6f76c2323c467758b8ffaa73b1f31adb28642616279536e656b",
+    "orgName": "BabySNEK",
+    "orgUrl": "https://babysnek.io/",
+    "tokenAddress": "931c7c94afa5afe4d86fe6f76c2323c467758b8ffaa73b1f31adb28642616279536e656b",
+    "tokenColorTheme": 12,
+    "tokenIcon": "/static/tokens/BABYSNEK.png",
```

Comment:
should use svg file

Reply:
Here, the switch to PNG image format were suggest by @kenblaue and @pepebndc :
![Screenshot_20241029_122111](https://github.com/user-attachments/assets/a3294b2a-aceb-4739-a179-6a578cb0159d)
![Screenshot_20241029_122035](https://github.com/user-attachments/assets/760369c2-4559-4b54-a7d0-79946e8b56d7)
![Screenshot_20241029_122047](https://github.com/user-attachments/assets/d45301be-35ad-4378-a134-c9c42c79cacc)